src/SprykerEco/Zed/Adyen/Business/Handler/Redirect/OnlineTransferRedirectHandler.php

```diff
@@ -20,6 +20,11 @@
 
 class OnlineTransferRedirectHandler implements AdyenRedirectHandlerInterface
 {
+    /**
+     * @var string
+     */
+    protected const RESULT_CODE_REFUSED = 'Refused';
```

Comment:
Can we use `AdyenConfig::ADYEN_PAYMENT_STATUS_REFUSED` instead?

Reply:
Please, remove unused `RESULT_CODE_REFUSED` constant from `OnlineTransferRedirectHandler`.